src/Text/Pandoc/App/CommandLineOptions.hs

```diff
@@ -439,7 +439,14 @@ options =
 
     , Option "" ["self-contained"]
                  (NoArg
-                  (\opt -> return opt { optSelfContained = True }))
+                  (\opt -> do
+                    deprecatedOption "--self-contained" "use `--embed-resources --standalone`"
```

Comment:
Let's omit the `` ` `` (backticks), since we don't use them on other deprecation option warnings.